source/views/Extension/Views/ImportWallet/Steps/Step2.jsx

```diff
@@ -17,11 +24,26 @@ const Step2 = ({ userPemFile }) => {
   const [walletName, setWalletName] = useState(""); // add deafault wallet name, for example the next wallet not used number
   const [currentEmoji, setCurrentEmoji] = useState("😎"); // add default emoji, not used in other wallets
   const [openEmojiSelector, setOpenEmojiSelector] = useState(false);
+  const [importedPrincipal, setImportedPrincipal] = useState();
+  const [detailsModal, setDetailsModal] = useState(false);
```

Comment:
I'd call it `detailsModalOpen` for clarity

Reply:
Changed it 👍